examples/cryptocurrency-advanced/backend/src/main.rs

```diff
@@ -12,12 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use exonum::helpers::fabric::NodeBuilder;
 use exonum_cryptocurrency_advanced as cryptocurrency;
+use exonum_parameters::run_node;
 
 fn main() {
     exonum::helpers::init_logger().unwrap();
-    NodeBuilder::new()
-        .with_service(cryptocurrency::CryptocurrencyService)
-        .run()
+    run_node(vec![Box::new(cryptocurrency::CryptocurrencyService) as Box<_>])
```

Comment:
What do you think about this kind of an interface? It looks a little verbose as for me.

Reply:
It'd be good to get rid of an explicit conversion. Is it really necessary here ?

Reply:
Yes, it is necessary.

I decided to use the old interface with `NodeBuilder` - it looks much more nice to me.